tests/app/views/test_buyers.py

```diff
@@ -587,6 +587,77 @@ def test_404_if_framework_status_is_not_live_(self, data_api_client):
 
 
 @mock.patch('app.buyers.views.buyers.data_api_client')
+class TestPublishBrief(BaseApplicationTest):
+    def test_publish_brief(self, data_api_client):
```

Comment:
Would be nice to have some tests on the GET route too - e.g. check publish button is there if brief is complete, isn't there if there are still unanswered questions.